test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -103,60 +103,50 @@ class VirtualEnvTest extends BasePipelineTest {
 
   @Test
   void newObjectUnix() {
-    if (JenkinsMocks.isUnix()) {
```

Comment:
why are we removing the checks here?

Reply:
Previously we only ran these tests on the respective platform that they were tested on. Now we run all tests on each platform, mocking `isUnix`. To tell you the truth, I'm not sure why we did this before. See 8e0a0b7af0c6f099587a045ff6cc4ea87c4be7ee.